risteys_elixir/assets/css/tailwind.css

```diff
@@ -333,34 +334,29 @@ header .actions a {
  * PHENOCODE PAGE
  */
 
-/* Risteys-link */
-#Risteys-link {
-    @apply mr-2; /* space between "Risteys" and "R7", set margin here no that it doesn't affect alignment of the submenu*/
-}
-
-/* <summary> */
+/* VERSION SELECTOR */
 #R-nav {
-    @apply ml-0; /* overwriting "header .actions a {@apply ml-6;}" to have "R7" closer to "Risteys" -> more on left -> subnav right below R7"*/ 
+    @apply ml-0; /* overwriting "header .actions a {@apply ml-6;}" to have "R7" closer to "Risteys" -> more on left -> subnav right below R7"*/
     list-style: none; /* remove arrow */
 }
 
 /* to remove arrow, this is used because Chromium-based browsers do not support list-style on the summary element */
-summary::-webkit-details-marker {display: none; }
+.subnav summary::-webkit-details-marker {display: none; }
```

Comment:
Here and following occurrences I added a more specific CSS selector. The styling here is specific to the version subnav, so it needs to not apply to other `<details>` or `<summary>` in the future.